indra/newview/skins/default/xui/en/panel_gltf_material.xml

```diff
@@ -339,6 +339,31 @@
      top_pad="5"
      name="emissive color"
      width="40" />
+    <text
+      type="string"
+      length="1"
+      follows="left|top"
+      height="10"
+      layout="topleft"
+      left_delta="0"
+      top_pad="5"
+      width="96"
+      name="emissive_strength_lbl">
+      Emissive Strength
+    </text>
+    <spinner
+      decimal_digits="3"
```

Comment:
I think this will set the limit to 999 or something, which could be quite low for this particular use case.

Reply:
It's _probably_ fine.  999 on white (or even like 0.1% grey) is gonna be pretty bright - even for HDR use cases.  There shouldn't be any clamping elsewhere beyond what's mandated by the spec in the file.

Reply:
Though this shouldn't be clamped to 1.  Fixing that.